web-client/src/views/TrialSessionDetails/TrialSessionDetails.tsx

```diff
@@ -75,6 +77,7 @@ export const TrialSessionDetail = connect(
           <TrialSessionInformation />
 
           {!formattedTrialSessionDetails.isCalendared && (
+            /* @ts-ignore: Tabs needs to be refactored to avoid type errors */
```

Comment:
Renaming file caused these type errors to register as "new" and thus fail the CI Typescript checks.